### PR TITLE
Add a tox target for 'woke' to check for inclusive naming in the charm

### PR DESCRIPTION
**Pull request by mthaddon**
Add a tox target for 'woke' to check for inclusive naming in the charm.

Since this is provided by a snap, we're creating a shell command to check you have the snap installed or error out, and then run the woke command.

This isn't yet hooked up to workflows, as this charm doesn't have those yet.

### CODE REVIEW

**Comment by mthaddon** [comment]
Marking as draft as this isn't quite ready for review yet, as it includes changes that remove containers, which is blocked on an external issue.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ enable it with `juju expose kubernetes-worker`.
 
 ## Usage
 
-You'll need version to be using Juju [version 2.9.0](https://discourse.charmhub.io/t/juju-2-9-0-release-notes/4525) or later, and Kubernetes version 1.19 or later.
+You'll need version to be using Juju [version 2.9.23](https://discourse.charmhub.io/t/roadmap-releases/5064) or later, and Kubernetes version 1.19 or later.
```
Comment (mthaddon) [inline README.md:19]: This change is unrelated, and has already landed in the repo.

File: metadata.yaml [hunk 2]
```diff
@@ -7,14 +7,8 @@ summary: |
...
-containers:
-  placeholder:
-    resource: placeholder-image
-
-resources:
-  placeholder-image:
-    type: oci-image
-    description: Docker image for placeholder - won't be used
+assumes:
+  - k8s-api
```
Comment (mthaddon) [inline metadata.yaml:12]: This change is unrelated and has already landed in the repo

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: .woke.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+rules:
```
Comment (jdkandersson) [inline .woke.yml:1]: I suppose the tool doesn't look in `pyproject.toml`?

File: tox.ini [hunk 4]
```diff
@@ -51,6 +52,11 @@ commands = flake8 src/ tests/ lib/
 deps =
     flake8==3.7.9
 
+[testenv:woke]
```
Comment (jdkandersson) [inline tox.ini:55]: Do you think this should be one of the checks in `lint` rather than a separate check? It seems to share many characteristics with a linting tool

**Review by jdkandersson — CHANGES_REQUESTED** [review]

File: .woke.yml (hunk 3 above)
Comment (mthaddon) [reply .woke.yml:1]: I don't think so, it's not a python-specific project.

File: tox.ini (hunk 4 above)
Comment (mthaddon) [reply tox.ini:55]: I agree it shares some characteristics, but it feels different enough to me that it warrants a different target. Happy to be overridden if I'm in the minority on that.

Comment (arturo-seijas) [reply tox.ini:55]: Maybe it will be appropriate to add this target the `envlist`, also, `lint`is missing

**Review by jdkandersson — DISMISSED** [review]

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by jdkandersson — APPROVED** [review]
